src/GameOptions/ui/NumericDisplayOptions.tsx

```diff
@@ -72,6 +77,15 @@ export const NumericDisplayPage = (): React.ReactElement => {
           <>If this is set all references to memory will use GiB instead of GB, in accordance with IEC 60027-2.</>
         }
       />
+      <OptionsSlider
+        label="Fractional Digits"
+        initialValue={Settings.fractionalDigits}
+        callback={handleFractionalDigitChange}
+        step={1}
+        min={0}
+        max={5}
+        tooltip={<>The number of digits of precision to display on small numbers.</>}
```

Comment:
```suggestion
        tooltip={<>The default number of decimal places to display on small numbers.</>}
```
"Decimal places"/"fractional digits" are not the same as "digits of precision". 12345.6789 to 3 fractional digits is 12345.679. To 3 digits of precision, it is 12300.

IMO also useful to mention "default", since lots of methods override this and thus won't be affected by this setting.

Reply:
Oh man you would not believe how long i was stuck on the wording of this, usually im fine with it but yesterday was not an english language day i guess. thanks for the improvement.